bridgeql/django/models.py

```diff
@@ -199,7 +202,11 @@ def _apply_opts(self):
                     self.qset = self._add_fields()
```

Comment:
2 lines up for the check query_has_properties
we can make use of model_config.get_properties to do this only for properties, and do normal operation for foreign_keys

Reply:
Resolved in the recent commit.